src/submodules/io/csv_reader.jl

```diff
@@ -0,0 +1,32 @@
+using CSV
+
+"""
+    populate_from_csv!(filename, model, agent_type, col_map; kwargs...)
```

Comment:
In Julia the typical convention to follow is that the argument that is mutated is first in the function.